src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesContractTests.cs

```diff
@@ -161,7 +162,7 @@ public static void ValidCFBFeedbackSizes(int feedbackSize)
             }
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
```suggestion
        [Theory]
```
There is no condition here

Reply:
I thought that in order to utilize `SkipTestException`, the `Theory` would need to be a `ConditionalTheory` to allow for one of the test cases to not be ran. Is this not the case?

Reply:
Ah ok, I did not know that.